sklearn/metrics/_dist_metrics.pxd.tp

```diff
@@ -5,30 +5,26 @@ implementation_specific_values = [
     #
     #       name_suffix, INPUT_DTYPE_t, INPUT_DTYPE
     #
+    #
     # On the first hand, an empty string is used for `name_suffix`
-    # for the float64 case as to still be able to expose the original
-    # float64 implementation under the same API, namely `DistanceMetric`.
+    # for the 64bit case as to still be able to expose the original
+    # 64bit implementation under the same API, namely `DistanceMetric`.
```

Comment:
I think we preferred to use `float64` over `64bit` because this makes explicit that we are talking about data and not CPU architecture.

What do you think?